### PR TITLE
Fixed major bug in CUR sample orthogonalization

### PR DESCRIPTION
**Pull request by rosecers**
The sample orthogonalizer for CUR was not properly orthogonalizing any samples that were after the selected sample.

### CODE REVIEW

**Review by Luthaf — CHANGES_REQUESTED** [review]
The code looks good, but this is a good opportunity to add tests checking that sample are properly orthogonalized after calling the functions. In fact, every bug fix should come with an associated test!

File: skcosmo/selection/orthogonalizers.py [hunk 1]
```diff
@@ -12,6 +12,16 @@
 import numpy as np
 
 
+def X_orthogonalizer(X_proxy, c):
```
Comment (Luthaf) [inline skcosmo/selection/orthogonalizers.py:15]: Also, if this is a public function, it should have a bit of documentation

File: tests/selection.py [hunk 2]
```diff
@@ -1,10 +1,18 @@
 import unittest
-import skcosmo
 from sklearn.datasets import load_boston
+import numpy as np
+
+from skcosmo.selection import FeatureCUR, FeatureFPS, SampleCUR, SampleFPS
+
+zero = 1e-8
```
Comment (Luthaf) [inline tests/selection.py:7]: Could you use `SCREAMING_SNAKE_CASE` for global/constant values? Also, this is usually called `EPSILON` from the C `FLT_EPSILON`/`DBL_EPSILON` macros.

Comment (rosecers) [reply tests/selection.py]: Done! Snakes are screaming.

**Review by Luthaf — APPROVED** [review]

**Pull request merged**